### PR TITLE
grails  build scripts

### PR DESCRIPTION
**Pull request by sapnashukla28**
**Package name**: grails
**Url**: https://github.com/grails/grails-core.git
**Version mentioned**: 3.3.12

The build script is however prepared for a default version **5.0.2** **(latest released version)**

**For 3.3.12:
On power setup :**
On running the gradle test the execution gets blocked at the following test, for several hours:
:grails-core

**On intel setup**
the given tests failing with following error:
```
org.grails.compiler.injection.DefaultDomainClassInjectorSpec > initializationError FAILED
    java.lang.ClassFormatError
[Thread-4] INFO org.springframework.context.annotation.AnnotationConfigApplicationContext - Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@3faae458: startup date [Fri Dec 10 08:25:29 UTC 2021]; root of context hierarchy

170 tests completed, 1 failed, 1 skipped
:grails-core:test FAILED

FAILURE: Build failed with an exception.
```

**For 5.0.2**
the commands execute completely without failure.

Thanks,
Sapna

### CODE REVIEW

File: g/grails/grails_ubi8.5.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# -----------------------------------------------------------------------------
+#
+# Package		: grails
+# Version		: v5.0.2
+# Source repo	: https://github.com/grails/grails-core.git
+# Tested on	: ubi 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapna Shukla <Sapna.Shukla@ibm.com>
```
Comment (kandarpamalipeddi) [inline g/grails/grails_ubi8.5.sh:8]: Please add `Language` field in Script header.

File: g/grails/grails_ubi8.5.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+export WORK_DIR=$HOME_DIR/$PACKAGE_NAME
+cd $WORK_DIR
+git checkout $PACKAGE_VERSION
+
+export HOME=/home/tester
+wget https://services.gradle.org/distributions/gradle-6.9.1-bin.zip -P /tmp && unzip -d /home/tester/gradle -f /tmp/gradle-6.9.1-bin.zip
+export GRADLE_HOME=/home/tester/gradle/gradle-6.9.1/
+rm -rf /tmp/gradle-6.9.1-bin.zip
+export PATH=${GRADLE_HOME}/bin:${PATH}
```
Comment (kandarpamalipeddi) [inline g/grails/grails_ubi8.5.sh:39]: We are using `./gradlew` at Line 42. Hence that script should handle setting gradle related setup.
Is there any reason for doing it explicitly ?

File: g/grails/grails_ubi8.5.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+	
+if ! ./gradlew install; then
+	echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+	echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+	exit 1
+fi
+
+if ! gradle test; then
```
Comment (kandarpamalipeddi) [inline g/grails/grails_ubi8.5.sh:48]: Can't we use `./gradlew test` here ?

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: g/grails/grails_ubi8.5.sh (hunk 1 above)
Comment (sapnashukla28) [reply g/grails/grails_ubi8.5.sh:9]: Added.

File: g/grails/grails_ubi8.5.sh (hunk 2 above)
Comment (sapnashukla28) [reply g/grails/grails_ubi8.5.sh]: Yes, re-ran the script after removing the Gradle setup details, working correctly.
Thanks.

File: g/grails/grails_ubi8.5.sh (hunk 3 above)
Comment (sapnashukla28) [reply g/grails/grails_ubi8.5.sh]: Yes replaced it with ./gradlew test.
Working correctly.

Thanks.

**Review by kandarpamalipeddi — APPROVED** [review]

**Pull request merged**